src/storage/config.rs

```diff
@@ -109,3 +112,51 @@ impl BlockCacheConfig {
         Some(Cache::new_lru_cache(cache_opts))
     }
 }
+
+pub const DEFAULT_GC_RATIO_THRESHOLD: f64 = 1.1;
+pub const DEFAULT_GC_BATCH_KEYS: usize = 512;
+const DEFAULT_GC_MAX_WRITE_BYTES_PER_SEC: u64 = 5 * MB;
```

Comment:
I think the default value should no limit?

Reply:
Okay